### PR TITLE
Adding a jdk 17 image for openshift-knative java-based Data-Plane builds

### PR DESCRIPTION
**Pull request by matzew**
Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>

### CODE REVIEW

File: core-services/image-mirroring/supplemental-ci-images/mapping_supplemental_ci_images_openshift [hunk 1]
```diff
@@ -5,4 +5,6 @@ registry.access.redhat.com/rhscl/nodejs-10-rhel7:latest registry.ci.openshift.or
 #docker.io/ansible/ansible-runner:latest registry.ci.openshift.org/openshift/ansible-runner:latest
 #docker.io/centos/ruby-25-centos7:latest registry.ci.openshift.org/openshift/release:jdk-8
 #docker.io/jboss/base-jdk:8 registry.ci.openshift.org/openshift/release:ruby-25
-quay.io/prometheus/prometheus:main registry.ci.openshift.org/ci/prometheus:latest
\ No newline at end of file
+quay.io/prometheus/prometheus:main registry.ci.openshift.org/ci/prometheus:latest
```
[the remark this answers is not in the document]
Comment (matzew) [reply core-services/image-mirroring/supplemental-ci-images/mapping_supplemental_ci_images_openshift:8]: @hongkailiu I did not change anything there - I just added a new line feed, to add more images in line `9` and `10`

**Comment by matzew** [comment]
@hongkailiu thanks for the two suggestions - I was not sure I can actually do (e.g. being allowed to). Greatly appreciated.

As far as the prometheus goes: I have no I idea if that is still relevant or not, I just added two images below it

**Comment by matzew** [comment]
@hongkailiu related question: Should I move this to our own `openshift-knative` file? For better separation?

**Comment by petr-muller** [comment]
/uncc 
I'll leave this one for Hongkai

**Comment by matzew** [comment]
/close

looks like atm we do actually not need to stash these images around. 

If that might change, I will reopen this - thanks for the help on the subject

**Pull request closed without merging**